### ISSUE TITLE
Conversations 6.0.4 (December 28, 2023) twilio updated their android sdk

### ISSUE THREAD

**Issue by nikubesliu** [jedusei/Twilio.Conversations.Xamarin#19]
Hi, looks like twilio had a bug in their android sdk and they fixed it with latest release. Now when I try to create a new client my app is crashing. Could you please take a look? 
https://www.twilio.com/docs/conversations/android/changelog#conversations-604-december-28-2023

**Comment by NishiokaTakeo** [comment]
Yes. It looks calling `IConversationsClient.CreateAsync()` fails sometimes. I am considering to upgrade 6.0.1 at least but currently not sure how I can build successfully on this repos.

Does anyone get success to build the project?

![image](https://github.com/user-attachments/assets/3a6ce445-334c-4d18-af6d-cf5b2258b415)
